lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -75,6 +99,10 @@ def data_dir_empty?
       false
     end
 
+    def relabel_postgresql_dir
```

Comment:
It looks like you removed the caller, but not this method definition. I think this can go too.